authui/src/authflowv2/components/settings-button.css

```diff
@@ -0,0 +1,79 @@
+    --settings-btn__font-family: var(--typography-label-large__font-family);
+    --settings-btn__font-size: var(--typography-label-large__font-size);
+    --settings-btn__line-height: var(--typography-label-large__line-height);
+    --settings-btn__letter-spacing: var(
+      --typography-label-large__letter-spacing
+    );
+    --settings-btn__font-weight: var(--typography-label-large__font-weight);
+    --settings-btn__ring-width: 1.5px;
+    --settings-btn__ring-color: var(--color-neutral-100);
+    --settings-btn__py: var(--settings-btn__font-size);
+
+    --settings-btn__bg-color--hover: var(--color-neutral-100);
+    --settings-btn__text-color--hover: var(--settings-btn__text-color);
+    --settings-btn__ring-width--hover: var(--settings-btn__ring-width);
+    --settings-btn__ring-color--hover: var(--settings-btn__ring-color);
+
+    --settings-btn__bg-color--active: var(--color-neutral-100);
+    --settings-btn__text-color--active: var(--settings-btn__text-color);
+    --settings-btn__ring-width--active: var(--settings-btn__ring-width);
+    --settings-btn__ring-color--active: var(--settings-btn__ring-color);
+
+    --settings-icon__font-size: 1.5rem; /* 24 / 16 = 1.5 */
+
+    --settings-btn__bg-color-alternative: var(--color-neutral-200);
+  }
+
+  :root.dark {
+    --settings-btn__text-color: var(--color-neutral-100);
+    --settings-btn__ring-color: var(--color-neutral-500);
+    --settings-btn__bg-color--hover: var(--color-neutral-500);
+    --settings-btn__bg-color--active: var(--color-neutral-500);
+
+    --settings-btn__bg-color-alternative: var(--color-neutral-200);
+  }
+
+  .settings-btn {
+    @apply text-center;
+    @apply flex justify-center items-center;
+
+    background-color: var(--settings-btn__bg-color);
+    font-family: var(--settings-btn__font-family);
+    font-size: var(--settings-btn__font-size);
+    line-height: var(--settings-btn__line-height);
+    letter-spacing: var(--settings-btn__letter-spacing);
+    font-weight: var(--settings-btn__font-weight);
+    padding: var(--settings-btn__py) 0;
+    color: var(--settings-btn__text-color);
+
+    &:hover {
+      background-color: var(--settings-btn__bg-color--hover);
+      color: var(--settings-btn__text-color--hover);
+      @apply ring-inset
+        ring-[length:var(--settings-btn\_\_ring-width--hover)]
+        ring-[color:var(--settings-btn\_\_ring-color--hover)];
+    }
+
+    &:active {
+      background-color: var(--secondary-btn__bg-color--active);
+      color: var(--secondary-btn__text-color--active);
+      @apply ring-inset
+        ring-[length:var(--secondary-btn\_\_ring-width--active)]
+        ring-[color:var(--secondary-btn\_\_ring-color--active)];
+    }
+  }
+}
+
+.settings-btn__icon {
+  font-size: var(--settings-icon__font-size);
+}
+
+.settings-btn__alternative_icon {
+  font-size: var(--settings-icon__font-size);
+  color: var(--settings-btn__bg-color-alternative);
+}
```

Comment:
We use reuse primary button and secondary button instead of introducing settings button